experimental/PropertyDDS/packages/property-dds/src/test/rebasing.spec.ts

```diff
@@ -432,6 +432,7 @@ describe("PropertyDDS", () => {
                                 testString +=
                                     "await opProcessingController.ensureSynchronized();\n";
                             }
+                            console.log(testString);
```

Comment:
```suggestion
```

Reply:
@ChumpChief can you shed some light on what the testString is supposed to be doing here? We build it up over the course of the test run, but don't do anything with it.

Reply:
Looks like it gets used on L156?

Reply:
Oh lol my bad, I see this is a different test now :-P  Prob just remove it, I'm guessing it was intended for similar use but never got finished up.

Reply:
Sounds good. Will do.